ztp/gitops-subscriptions/argocd/SNOExpansion.md

```diff
@@ -0,0 +1,483 @@
+# SNO expansion #
+
+SNO expansion is a scenario where initially a SNO is deployed into production, and at a later date the cluster size is increased by the addition of a worker node.
+The resulting cluster will contain two nodes:
+- Original SNO serving as the control plane for itself and the additional node
+- One additional node without any control plane components (controlled by the SNO).
+
+This transition incurs zero downtime on the original SNO node.
+Addition of workers to an SNO cluster will increase CPU resources used by the original (master) node for control plane functions it performs. Although there is no hard limit on amount of the additional workers that can be added, worker addition must come with re-evaluation of reserved CPU allocation on the master. 
+
+## Prerequisites ##
+1. A cluster installed and configured using the GitOps ZTP fow, as described in [GitOps ZTP flow](README.md)
+1. ACM 2.6 (or above) with MultiClusterHub created and configured, running on OCP 4.11 (or above) bare metal cluster
+1. Central Infrastructure Management configured as described in [Advanced Cluster Management documentation](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/clusters/managing-your-clusters#enable-cim)
+1. TALM and openshift-gitops operators are installed and configured on the hub cluster
+1. The DNS serving the cluster is configured to properly resolve internal API endpoint `api-int.<cluster_name>.<base_domain>`. Please check [Requirements for installing OpenShift on a single node](https://docs.openshift.com/container-platform/4.11/installing/installing_sno/install-sno-preparing-to-install-sno.html#install-sno-requirements-for-installing-on-a-single-node_install-sno-preparing) for more details.
+1. ZTP container version is 4.12 or above
+1. Initial state:
+   1. the spoke cluster is managed by the hub.
+   2. ArgoCD applications are synchronized and all policies are compliant on the spoke cluster.
+
+## Procedure ##
+### Deploying a worker node ###
+1. Assuming your cluster was deployed using [this SiteConfig manifest](example/siteconfig/example-sno.yaml), add your new worker node to `spec.clusters['example-sno'].nodes` list, for example:
```

Comment:
Can this also be added "day-0" if the user wants to initially deploy SNO+worker?

Reply:
No, this is not a supported flow.